packages/react-accordion/src/components/Accordion/useAccordionContext.ts

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { createDescendantContext, useDescendant, useDescendantsInit } from '@reach/descendants';
+import { AccordionContext, AccordionDescendant, AccordionOpen, AccordionState } from './Accordion.types';
+import { useConst } from '@fluentui/react-utilities';
+
+export const accordionDescendantContext = createDescendantContext<AccordionDescendant>('AccordionDescendantContext');
+
+export const accordionContext = React.createContext<AccordionContext>(undefined!);
+
+export function useCreateAccordionContext(state: AccordionState) {
+  const { open, multiple, collapsible, onToggle } = state;
+  const isControlled = useConst(typeof open !== 'undefined');
+  const [descendants, setDescendants] = useDescendantsInit<AccordionDescendant>();
+  const [openItems, setOpenItems] = React.useState<AccordionOpen>(() => initializeOpenItems(state));
```

Comment:
I would expect there usage of `useControllableValue()` (from `react-utilities`), it have been already used in `Menu`, for example. We can sync in offline, if you will have troubles with its integration 😉

Reply:
I've actually had a previous version using the `useControllableValue` but since @ling1726  was doing some changes on that I opted for not using until everything is merged.